openeogeotrellis/job_tracker_v2.py

```diff
@@ -16,6 +16,8 @@
 from typing import Callable, List, NamedTuple, Optional, Union
```

Comment:
The new implementation rendered some imports unused so they can be removed.

Reply:
Removed the unused imports.